### DISCUSSION TITLE
All the newly exported project are not working

### DISCUSSION THREAD

**Discussion by photogyulai** [splinetool/react-spline#119] [Q&A]
**Hello there!** 
Could anybody help? Recently i exported an (vanila js) version, html/js/scene.splinecode trio.
It is on FTP now. On the frontend i link it like this:
```
<iframe width="100%" height="550" src="https://www.mypage.com/myfolder/index.html
</iframe>
```
All of my older project work fine like this, but since some week ago the method stopped working. So the old projects are fine but now i does not matter wich spline project i try it did not work.

What am i missing?
![2023-02-07_1](https://user-images.githubusercontent.com/23648105/221371906-ce2dea8d-8a62-45e4-994c-00ff69f30d83.jpg)

**Comment by marcofugaro** [comment]
You need to update the `runtime.js` file, the latest one can be found here:

https://unpkg.com/@splinetool/runtime@0.9.238/build/runtime.js

**Reply by photogyulai** [reply]
thanks Marco! Unfortunately did not work.
1. I downloaded the "runtime.js" file you mentioned. And i replaced this file with the runtime.js on the server.
2. I tried 3 browsers with a heavy "Ctrl+F5" usage (refresh)
No change.  

Here you can see:
https://www.fotobetyar.hu/it-fejlesztesek-2000/

The "empty upper area" under the "EGY KIS 3D FOTÓBETYÁR ILLUSZTÁCIÓ NEKED" text. Thats where this new spline should be displayed. But it isnt.

If you scroll down >> there are an older spline project. It works fine.

**Reply by marcofugaro** [reply]
You have to download the `opentype.js` file as well, it says in your console.

You can find it in this directory: https://unpkg.com/browse/@splinetool/runtime@0.9.238/build/

Next time, you can click the "Download Bundled ZIP" button and it will download all needed files for you in a .zip.

**Reply by photogyulai** [reply]
yes. Sorry, i guess i was just accidentaly deleting the opentype.js file during some "try to get working" process.

So yes >> i usually click "Downlaod bundled zip". Extract it, and upload the extracted content to FTP folder.

Still not working.   :-(

I tried with many of my newer projects. Not working. (the spline file name is "scene.splinecode" )
I tried with my older projects. Works fine. (the spline file name is "scene.spline" )

**Reply by photogyulai** [reply]
I did some test.
Currently i wait a couple of days becose:
1. Spline website sometimes available, sometimes dont.
2. If i export a project >> "export as zip" sometimes it comes with opentype.js file, sometimes i dont get any opentype.js file.
3. If i dont get any opentype.js file the uploaded project not works. That is for sure.
4. If i upload this file manually, sometimes it works, sometimes dont.
4. The projects often crashes during editing in Spline.
(sadly my experience are offten similar with Spline.)

**Reply by marcofugaro** [reply]
What is the error now?

**Reply by photogyulai** [reply]
Thanks Marco for the effort!
In one line: Basicaly all of my newer project are not working.
(usually i spend more time to get rid of x-y-z bugs, than i spent time on the 3d project. :-(
